src/renderer/controllers/torrent-list-controller.js

```diff
@@ -103,18 +103,73 @@ module.exports = class TorrentListController {
     }
   }
 
+  pauseAll ({filter, excluded}) {
```

Comment:
What about making this
```
prioritizeTorrent (infohash) {
```

...the filter arg is always just active torrents. Don't know if we need the ability to pause torrents by arbitrary JSON filter. The excluded arg is always an array containing a single infohash: the torrent we're playing.